### ISSUE TITLE
This is not tree-of-thoughts framework from the paper..

### ISSUE THREAD

**Issue by jakderrida** [kyegomez/tree-of-thoughts#34]
I want you to honestly consider the damage this fake repo had done to those, like me, that confused it for the real one and were hopelessly confused how, otherwise brilliant researchers, have produced such a negligently idiotic implementation of their own framework. Surely, for every star you got are dozens times more that thought the same, most of which remain decided to give up completely on the entire framework, either because they were horrified by the code upon seeing it or that they made the mistake of trying to make it work. Not one person that read the paper and confused your fake repo for that of the paper will be more informed.

Imagine all those researchers excited to test the real code and use it in their own research only to dismiss the Authors and framework as the makers of this garbage you had ChatGPT autogenerate. All those citations and breakthroughs now taken away just so an attention-seeking troll could get a few stars and forks on Github. Unfathomable.

**Comment by jakderrida** [comment]
Real repo is here, by the way:
https://github.com/ysymyth/tree-of-thought-llm

**Comment by kyegomez** [comment]
We provided the code and implementation before the authors did, and we may have made some mistakes but we provided something when there was nothing.

People wanted to get their hands on tree of thoughts we implemented it, it's very simple.

You seem to just assume the worst in people, why?

Why not remain hopeful?

And this isn't a fake repository this is an open source implementation of an repository based on an paper that had not released their code.
